### ISSUE TITLE
Plugin crash after opening project: java.lang.NoClassDefFoundError: com/intellij/psi/impl/source/tree/java/PsiLiteralExpressionImpl

### ISSUE THREAD

**Issue by jgjl** [giraud/reasonml-idea-plugin#333]
plugin version: 0.101-2021.1

### Description

Windows 10 with CLion, Ocaml runs directly on Windows (no WSL).
I started Clion with the ReasonML plugin installed and opened my Ocaml project: https://github.com/jgjl/ocaml-netaddr/tree/new_parser_merge

This is the result:
```
java.lang.NoClassDefFoundError: com/intellij/psi/impl/source/tree/java/PsiLiteralExpressionImpl
	at com.reason.lang.core.psi.PsiLiteralExpression$StringLiteralEscaper.decode(PsiLiteralExpression.java:52)
	at com.intellij.httpClient.http.security.HttpUrlsUsageInspection.decodeLiteralValue(HttpUrlsUsageInspection.kt:198)
	at com.intellij.httpClient.http.security.HttpUrlsUsageInspection.getLiteralFragments(HttpUrlsUsageInspection.kt:133)
	at com.intellij.httpClient.http.security.HttpUrlsUsageInspection.access$getLiteralFragments(HttpUrlsUsageInspection.kt:26)
	at com.intellij.httpClient.http.security.HttpUrlsUsageInspection$buildVisitor$1.visitElement(HttpUrlsUsageInspection.kt:69)
	at com.intellij.psi.impl.source.tree.LeafPsiElement.accept(LeafPsiElement.java:208)
	at com.intellij.codeInspection.InspectionEngine.acceptElements(InspectionEngine.java:65)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.lambda$visitRestElementsAndCleanup$7(LocalInspectionsPass.java:346)
	at com.intellij.util.AstLoadingFilter.lambda$toComputable$2(AstLoadingFilter.java:174)
	at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:132)
	at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:121)
	at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:111)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.lambda$visitRestElementsAndCleanup$10(LocalInspectionsPass.java:346)
	at com.intellij.concurrency.ApplierCompleter.execAndForkSubTasks(ApplierCompleter.java:136)
	at com.intellij.concurrency.ApplierCompleter.execAndForkSubTasks(ApplierCompleter.java:149)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1096)
	at com.intellij.concurrency.ApplierCompleter.lambda$wrapInReadActionAndIndicator$1(ApplierCompleter.java:92)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:688)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:634)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:64)
	at com.intellij.concurrency.ApplierCompleter.wrapInReadActionAndIndicator(ApplierCompleter.java:104)
	at com.intellij.concurrency.ApplierCompleter.lambda$compute$0(ApplierCompleter.java:83)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:167)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:178)
	at com.intellij.concurrency.ApplierCompleter.compute(ApplierCompleter.java:83)
	at java.base/java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:746)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
Caused by: java.lang.ClassNotFoundException: com.intellij.psi.impl.source.tree.java.PsiLiteralExpressionImpl PluginClassLoader(plugin=PluginDescriptor(name=ReasonML, id=reasonml, descriptorPath=plugin.xml, path=~\AppData\Roaming\JetBrains\CLion2021.1\plugins\reasonml-idea-plugin, version=0.101-2021.1, package=null), packagePrefix=null, instanceId=86, state=active)
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:215)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	... 31 more
```

**Comment by giraud** [comment]
I reproduced it. thnaks for the report.

**Comment by jgjl** [comment]
@giraud Thank you! 
I just tried to test the previous release, but it is not compatible with my version of Clion: "Plugin 'ReasonML' (version '0.101-2020.3') is not compatible with the current version of the IDE, because it requires build 203.* or older but the current build is CL-211.7442.42"

**Comment by giraud** [comment]
I think it'll take some time to clean up.

**Comment by ktrzos** [comment]
I'm also experiencing this issue.

**Comment by giraud** [comment]
fixed in 0.102.1
